.github/workflows/check.yml

```diff
@@ -1,6 +1,8 @@
 name: check
 on:
   push:
+    branches: [master, 'test-me-*']
+    tags:
```

Comment:
Empty tags mean run on all tags? Or is it do not run it on tags?

Reply:
all tags